pollux/lib/anoncreds/src/main/scala/io/iohk/atala/pollux/anoncreds/AnoncredLib.scala

```diff
@@ -138,19 +138,23 @@ object AnoncredLib {
   type SchemaId = String
   type CredentialDefinitionId = String
 
+  // TODO FIX
+  // [info] uniffi.anoncreds.AnoncredsException$CreatePresentationException: Create Presentation: Error: Error: Invalid structure
+  // [info] Caused by: Predicate is not satisfied
+
   def createPresentation(
       presentationRequest: PresentationRequest,
-      credentials: Seq[Credential],
+      credentialRequests: Seq[CredentialAndRequestedAttributesPredicates],
```

Comment:
would it be a good naming:
`presentationRequest`: AnonCredsPresentationRequest?